graylog2-server/src/main/java/org/graylog/security/certutil/CertRenewalServiceImpl.java

```diff
@@ -203,13 +204,19 @@ public void initiateRenewalForNode(final String nodeId) {
     @Override
     public List<DataNode> findNodes() {
         final Map<String, Node> activeDataNodes = nodeService.allActive(Node.Type.DATANODE);
-        return activeDataNodes.values().stream().map(node -> {
+        return addProvisioningInformation(activeDataNodes.values());
+    }
+
+    @Override
+    public List<DataNode> addProvisioningInformation(Collection<Node> nodes) {
```

Comment:
Could we rename the method to signalize that it extends the nodes collection and is not adding anything to the database?